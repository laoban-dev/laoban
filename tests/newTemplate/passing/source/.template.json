{
  "files": [
    {
      "file": ".npmrc",
      "target": ".npmrc"
    },
    {
      "file": ".template.json",
      "target": ".template.json"
    },
    {
      "file": "babel.config.json",
      "target": "babel.config.json"
    },
    {
      "file": "LICENSE.md",
      "target": "LICENSE.md"
    },
    {
      "target": "package.json",
      "file": "package.json",
      "type": "${}",
      "postProcess": "json"
    },
    {
      "file": "tsconfig.json",
      "target": "tsconfig.json"
    }
  ]
}