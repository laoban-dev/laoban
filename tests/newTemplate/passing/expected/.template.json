{
  "files": [
    {
      "file": ".npmrc",
      "target": ".npmrc"
    },
    {
      "file": "babel.config.json",
      "target": "babel.config.json"
    },
    {
      "file": "LICENSE.md",
      "target": "LICENSE.md"
    },
    {
      "target": "package.json",
      "file": "package.json",
      "type": "${}",
      "postProcess": "json"
    },
    {
      "file": "tsconfig.json",
      "target": "tsconfig.json"
    }
  ]
}